access/slackbot/Makefile

```diff
@@ -0,0 +1,72 @@
+VERSION=0.1.0-dev.1
+
+BUILDDIR ?= build
+BINARY = $(BUILDDIR)/slackbot
+GITTAG=v$(VERSION)
+ADDFLAGS ?=
+BUILDFLAGS ?= $(ADDFLAGS) -ldflags '-w -s'
+CGOFLAG ?= CGO_ENABLED=1
+
+OS ?= $(shell go env GOOS)
+ARCH ?= $(shell go env GOARCH)
+RELEASE_NAME=teleport-access-slackbot
+RELEASE=$(RELEASE_NAME)-$(GITTAG)-$(OS)-$(ARCH)-bin
+
+RELEASE_MESSAGE := "Building with GOOS=$(OS) GOARCH=$(ARCH)."
+
+VERSRC = version.go gitref.go
+
+export
+
+.PHONY: $(BINARY)
+$(BINARY):
+	GOOS=$(OS) GOARCH=$(ARCH) $(CGOFLAG) go build -o $(BINARY) $(BUILDFLAGS)
+
+clean:
+	@echo "---> Cleaning up build artifacts."
+	rm -rf $(BUILDDIR)
+	-go clean -cache
+	rm -rf $(RELEASE_NAME)
+	rm -rf *.gz
+	rm -rf *.zip
+	rm -f gitref.go
+
+.PHONY: release
+release: clean $(VERSRC) $(BINARY)
+	@echo "---> $(RELEASE_MESSAGE)"
+ifeq ("$(OS)", "windows")
+	$(MAKE) --no-print-directory release-windows
+else
+	$(MAKE) --no-print-directory release-unix
+endif
+
+.PHONY: release-unix
+release-unix:
+	@echo "---> Creating unix-like release archive."
+	mkdir $(RELEASE_NAME)
+	cp -rf $(BINARY) \
+		README.md \
+		CHANGELOG.md \
```

Comment:
Currently  breaks because `CHANGELOG.md` doesn't exist yet.  We haven't started doing versioning yet, so this is expected, but I prefer to keep master in a working state.  Might be worth adding a simple placeholder CHANGELOG for now.